tests/include/ut_test_common.h

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
? this shouldn't need to exist..

ut.h has this and therefore link against that.